src/tcgwars/logic/impl/gen4/GreatEncounters.groovy

```diff
@@ -2166,8 +2173,15 @@ public enum GreatEncounters implements LogicCardInfo {
         return basicTrainer (this) {
           text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card.\nDiscard up to 2 cards from your hand. If you discard 1 card, draw 3 cards. If you discard 2 cards, draw 4 cards."
           onPlay {
-          }
-          playRequirement{
+            assert my.deck : "Your deck is empty"
+            assert my.hand.size() > 1 : "You can't play this card"
```

Comment:
Should these assertions be playRequirements instead?

Reply:
Yes!

Reply:
Fixed